myServer/generator.js

```diff
@@ -0,0 +1,115 @@
+const faker = require('faker');
+const random = require('random');
+
+const fs = require('fs');
+
+// Faker + Media Asset Generator
+function generateFakeData(itemID) {
+  return {
+    id: itemID,
+
+    gif1: `https://sdc-media-data.s3-us-west-1.amazonaws.com/giphys/giphy+(${random.int(min=1, max=10)}).webp`,
+    gif2: `https://sdc-media-data.s3-us-west-1.amazonaws.com/giphys/giphy+(${random.int(min=1, max=10)}).webp`,
+    gif3: `https://sdc-media-data.s3-us-west-1.amazonaws.com/giphys/giphy+(${random.int(min=1, max=10)}).webp`,
+    image1: `https://sdc-media-data.s3-us-west-1.amazonaws.com/Images/download+(${random.int(min=1, max=20)}).jpg`,
+    image2: `https://sdc-media-data.s3-us-west-1.amazonaws.com/Images/download+(${random.int(min=1, max=20)}).jpg`,
+    image3: `https://sdc-media-data.s3-us-west-1.amazonaws.com/Images/download+(${random.int(min=1, max=20)}).jpg`,
+    storyT1: faker.fake('{{hacker.ingverb}} {{hacker.noun}}'),
+    storyT2: faker.fake('{{hacker.ingverb}} {{hacker.noun}}'),
+    storyT3: faker.fake('{{hacker.ingverb}} {{hacker.noun}}'),
+    storyT4: faker.fake('{{hacker.ingverb}} {{hacker.noun}}'),
+    storyT5: faker.fake('{{hacker.ingverb}} {{hacker.noun}}'),
+    storyText1: faker.hacker.phrase(),
+    storyText2: faker.hacker.phrase(),
+    storyText3: faker.hacker.phrase(),
+    storyText4: faker.hacker.phrase(),
+    storyText5: faker.hacker.phrase(),
+
+    riskT1: faker.hacker.noun(),
+    riskT2: faker.hacker.noun(),
+    riskT3: faker.hacker.noun(),
+    riskT4: faker.hacker.noun(),
+    riskT5: faker.hacker.noun(),
+    riskText1: faker.hacker.phrase(),
+    riskText2: faker.hacker.phrase(),
+    riskText3: faker.hacker.phrase(),
+    riskText4: faker.hacker.phrase(),
+    riskText5: faker.hacker.phrase(),
+
+    commitT1: faker.fake('{{hacker.ingverb}} {{hacker.noun}}'),
+    commitT2: faker.fake('{{hacker.ingverb}} {{hacker.noun}}'),
+    commitT3: faker.fake('{{hacker.ingverb}} {{hacker.noun}}'),
+    commitT4: faker.fake('{{hacker.ingverb}} {{hacker.noun}}'),

```

Comment:
Great technical use of reduce, however I suspect part of the reason your generator is taking so long is because you are building an object, and then reducing it to a single line of text. 

If you want to increase speed, I recommend you create a function that generates your data into a format appropriate for a csv.

Reply:
I will investigate further on this.